student/tests.py

```diff
@@ -164,3 +222,49 @@ def test_update_student_account_settings_with_valid_data(example_student_a, exam
     assert example_student_a_from_db.about == new_about
     # Test that untouched data hasn't changed
     assert example_student_a_from_db.educational_institution == example_student_a.educational_institution
+
+
```

Comment:
The tests bellow suffer from similar issues to the ones I already mentioned for the recruiter tests.